src/main/java/no/sikt/nva/model/dublincore/DcValue.java

```diff
@@ -79,6 +79,10 @@ public boolean isLanguage() {
         return Element.LANGUAGE.equals(this.element);
     }
 
+    public boolean isIsoLanguage() {
+        return Element.LANGUAGE.equals(this.element) && Qualifier.ISO.equals(this.qualifier);
```

Comment:
Heller ha en sjekk på om innholdet i value er et iso language istedet for å stole på dette.

Reply:
done

Reply:
Den er grei